src/java.base/share/classes/sun/security/pkcs12/PKCS12KeyStore.java

```diff
@@ -1431,7 +1434,7 @@ public synchronized void engineSetEntry(String alias, KeyStore.Entry entry,
     /*
      * Assemble the entry attributes
```

Comment:
Perhaps be a bit more specific here - "Populate the entry with additional attributes used by the implementation."